### ISSUE TITLE
Rendering of placeholders in edit view is broken

### ISSUE THREAD

**Issue by faebser** [divio/django-cms#3475]
I'm using the following structure in a project:
``` python
{% block content %}
        <article class="detail">
        {% placeholder intro %}
            <div class="article-content">
                {% placeholder content %}
                <div class="footnotes">
                    {% placeholder footnotes %}
                </div>
            </div>
        </div> <!-- closing of article-wrapper inside of intro !-->
        <aside id="sidebar" class="sidebar-detail">
            {% placeholder internal-links %}
            {% placeholder external-links %}
            {% placeholder ads %}
        </aside>
        </article>
{% endblock content %}
```
Sadly neither the placeholder content nor the placeholder footer show up in the edit view.

![placeholder_error_01](https://cloud.githubusercontent.com/assets/333862/4468399/69d59f04-48fd-11e4-9161-612ae1ca8ef0.png)

I think it is related to the css-styles of .article-content and #sidebar. Below are the rendered root-tags of the placeholders.

```html
<div class="cms_dragarea cms_dragarea-4" style="opacity: 1; top: -5px; left: -150px; width: 150px;"> <!-- placeholder content --> 
<div class="cms_dragarea cms_dragarea-8" style="opacity: 1; top: 223px; left: 750.5px; width: 710px;"> <!-- placeholder intro -->
<div class="cms_dragarea cms_dragarea-9" style="opacity: 1; top: -5px; left: -150px; width: 150px;"> <!-- placeholder footnotes -->

```

```css
.article-content {
margin-top: 60px;
position: relative;
padding-right: 25px;
clear: both;
}

#sidebar {
width: 188px;
float: left;
margin-top: -4px;
margin-right: -214px;
margin-left: 26px;
}
```

I'm using
Django==1.6.5
django-cms==3.0.3

**Comment by yakky** [comment]
Postponing after 3.1 as it requires rethinking of structure mode

**Comment by kakulukia** [comment]
hi .. i kinda have the same bug i think

i have a base template with placeholder1
and a default template with show_placeholder (which shows the header) and placeholder2

now when i switch to the structure view of any page based on the default template, the cms sometimes displays the placeholder1, but most of the times it does not and static/cms/js/modules/cms.structureboard.js fails because the dragarea is not rendered and thus the positioning of placeholders is fucked up
![bildschirmfoto 2015-07-03 um 14 28 22](https://cloud.githubusercontent.com/assets/96352/8499153/d4b9d6c6-218f-11e5-8e54-3c0137369e8b.png)

i found a quick fix tho - which even cant hurt having it in the vanilla code:
in cms.structureboard.js _resizeBoard() below
area = $('.cms_dragarea-' + id);
add this line:
if (area.length == 0) return;

this way the editor is usable again .. im just not able to edit stuff in the referenced header placeholder, but this i can do by switching to the header page .. so its still possible

**Comment by FinalAngel** [comment]
This has been addressed in 3.2 :)
